src/pages/404.jsx

```diff
@@ -0,0 +1,14 @@
+import Head from 'next/head';
+import Link from 'next/link';
+
+export default function ErrorPage() {
+    return (
+			<div>
+				<Head>
+					<title>404 Not Found</title>
+				</Head>
+				<h1>404 Not Found</h1>
+				<Link href='/'>Overview Page</Link>
```

Comment:
This line needs a small change. The way the NextJS `<Link> ` component is designed you need to give it an `<a>` tag as a child. Additionally to keep accessibility linters happy you need to give that `<a>` tag an `href` attribute that ultimately gets replaced by NextJS. So in summary this line should look something like this. 
```
<Link href='/' passHref><a href="replace"> Overview Page</a></Link>
``` 
This is one of those things thats specific to NextJS and you wouldn't know unless you've seen it before.

Reply:
We were actually just talking about that today! Thanks for catching it, I had forgotten it applied here :)

Reply:
Actually, I re-read the conversation and I think Jared said we don't need the a (according to Ben). Let me double check in the apprentice channel because I think Jared's was written without and I want to make sure that's correct and we are all consistent. Will post the question in the a.m. before I leave for CT.

Reply:
This is what I'm referring to:
![image](https://user-images.githubusercontent.com/49820744/194441749-8b84a604-4a15-4c79-b754-4d4a5211bff7.png)

Reply:
I also just ran the npm lint and it didn't seem to run into a problem:

![image](https://user-images.githubusercontent.com/49820744/194441922-81d4af48-56b2-456c-9825-3731e86da12d.png)

Reply:
I did some digging and it looks like starting with Next 13 this will be ok. [Here is where I found it](https://github.com/vercel/next.js/pull/36436). Sometimes you can find good things in a GitHub repo. Looking at `package.json` and this app is still running Next 12. I would not upgrade it here because I would consider that to be out of scope but if the project does get updated to Next 13 the links could be refactored then. Good work double checking when you were not sure about my requested changes. Code reviews are a conversation so discussion is good.

Reply:
I think this looks good! The file here still has some odd spacing, but when I pull it and open in my editor it's fine! 

GitHub: 
<img width="650" alt="Screen Shot 2022-10-07 at 12 26 12 PM" src="https://user-images.githubusercontent.com/69602589/194602338-58d8595a-f2cf-450e-bb81-d3ab85d21cdf.png">

Local: 
<img width="383" alt="Screen Shot 2022-10-07 at 12 26 41 PM" src="https://user-images.githubusercontent.com/69602589/194602383-e0999dcd-c837-4b44-9b48-230acfde7aae.png">


I tested it in Edge, so I checked that off. The only things I found in Axe were the things you already found, and we're addressing in the next cards. Way to go!

Reply:
Thanks Theo, I updated the links on this [commit 1826998](https://github.com/sparkbox/accessible-components/pull/9/commits/1826998451f350c1dca93c000421ca82e923a2cc) and explained why I needed to do so!